index.js

```diff
@@ -67,7 +67,12 @@ ipcMain.handle('dragOut', (_, value) => {
   })
 })
 
-if (!(app.getVersion() === store.get('lastInstalledAppVersion'))) {
+if (
+  !(app.getVersion() === store.get('lastInstalledAppVersion')) &&
```

Comment:
```suggestion
  app.getVersion() !== store.get('lastInstalledAppVersion') &&
```

Easier to reason about without the double logic